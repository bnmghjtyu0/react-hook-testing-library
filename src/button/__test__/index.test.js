import React from 'react'
import { render, fireEvent } from '@testing-library/react'
const { toMatchDiffSnapshot } = require('snapshot-diff');
import Button from '../index'

describe('Button', () => {
    it('snapshot', () => {
        const initialProps = {
            data: {
                name: 'Richard'
            }
        }
        const { container, debug, getByTestId, asFragment } = render(<Button {...initialProps} />)
        expect(container.firstChild).toMatchSnapshot();

    })
})