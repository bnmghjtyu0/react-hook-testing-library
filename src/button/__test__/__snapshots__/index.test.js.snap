// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button adf 1`] = `
<div
  data-testid="component-button"
>
  <div>
    <button
      data-testid="btn-toggle"
    >
      修改
    </button>
    <span>
      Richard
    </span>
  </div>
</div>
`;
